src/PeopleAdapter.js

```diff
@@ -68,4 +68,15 @@ export default class PeopleAdapter extends WebexAdapter {
   getPerson(ID) {
     return throwError(new Error('getPerson(ID) must be defined in PeopleAdapter'));
   }
+
+  /**
+   * Returns an observable that emits people list
+   *
+   * @param {string} query Search query
+   * @returns {external:Observable.<Person[]>} Observable that emits person list based on search query
+   * @memberof PeopleAdapter
+   */
+  getPeopleList(query) {
```

Comment:
I think the name can be more action-oriented

```suggestion
  searchPeople(query) {
```